pages/demo/blocks.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
```

Comment:
It would be better off deleting the file and moving the demonstration icons to storybook (Icon.stories.tsx).

Reply:
It was in the storybook (Icon.stories.tsx). Demo page has been removed.